data/player/Admin.json

```diff
@@ -1,62 +1,34 @@
 {
-  "attributes": [
-    [
-      "health",
-      {
-        "delta": 0,
-        "base": 100
-      }
-    ],
-    [
-      "mana",
-      {
-        "delta": 0,
-        "base": 200
-      }
-    ],
-    [
-      "energy",
-      {
-        "delta": 0,
-        "base": 200
-      }
-    ],
-    [
-      "strength",
-      {
-        "delta": 0,
-        "base": 20
-      }
-    ],
-    [
-      "intelligence",
-      {
-        "delta": 0,
-        "base": 20
-      }
-    ],
-    [
-      "wisdom",
-      {
-        "delta": 0,
-        "base": 20
-      }
-    ],
-    [
-      "dexterity",
-      {
-        "delta": 0,
-        "base": 20
-      }
-    ],
-    [
-      "constitution",
-      {
-        "delta": 0,
-        "base": 20
-      }
-    ]
-  ],
+  "attributes": {
```

Comment:
I'm sure you've noticed this by now, I've changed some of the JSON.stringify(data) to JSON.stringify(data, null, 2) for better json formatting (so we can read it easier)